.gitignore

```diff
@@ -1,3 +1,9 @@
+/*
+/!*.*
+/!*/
```

Comment:
@kieferro Yup the idea is just to ignore all binary files without an extension - basically just so I can put a stockfish engine in the directory for testing. I used the first part of this answer here: https://stackoverflow.com/a/25592735. The answer gave this solution without the '/' prefix, so if needed we can do that - ignoring all binary files in all the subfolders makes sense anyway, since they're never used I think.

Would you mind sending a screenshot of how it doesn't work for you btw?

Reply:
Okay, that sounds reasonable and with the new changes it works now. Before that I had the problem that everything was just ignored. So when I added a folder that contains a python file (with extension) and then tried to add it, git tells me that this folder is ignored:
> The following paths are ignored by one of your .gitignore files:
> new_folder
> hint: Use -f if you really want to add them.
> hint: Turn this message off by running
> hint: "git config advice.addIgnoredFile false"